client/blocks/login/login-header.tsx

```diff
@@ -79,6 +78,16 @@ export function getHeaderText(
 			clientName = 'Blaze Pro';
 		} else if ( isA4AOAuth2Client( oauth2Client ) ) {
 			clientName = 'Automattic for Agencies';
+		} else if ( isWCCOM ) {
+			if ( twoFactorEnabled ) {
+				headerText = translate( 'Authenticate your login' );
+			} else if ( currentQuery.lostpassword_flow ) {
+				headerText = translate( 'Log in to your account' );
+			} else if ( showContinueAsUser ) {
+				headerText = translate( 'Log in to your account' );
+			}
```

Comment:
Just curious why we need to keep these 🤔

Reply:
I thought they do something, but they don't work. So removed them now.